ci/.travis.yml

```diff
@@ -12,29 +12,14 @@ before_install:
 bundler_args: "--binstubs --standalone --without documentation --path ../bundle"
```

Comment:
To be honest you can delete this file, we won't be using it after the release of RSpec 4, and Jruby is being worked on elsewhere.

Reply:
I could, but https://github.com/rspec/rspec-dev/blob/d27313db806a4a033208e4ac6f637f9009417bfb/ci/script/functions.sh#L82

```
bundle_install_flags=`cat .travis.yml | grep bundler_args | tr -d '"' | grep -o " .*"`
```

I tried to keep this PR focused to its title, "Drop CI support for Ruby < 2.3", not to upset code reviewers 😄 

Ok to adjust `ci/script/functions.sh` and inline those bundler flags? They don't differ across repos `"--binstubs --standalone --without documentation --path ../bundle"`.

I'd rather keep `.travis.yml` for now if you don't mind. Because some specs pass on all MRIs, but fail on JRuby on CI. And if my memory doesn't let me down, I couldn't reproduce them locally.
Again, `.travis.yml` will be removed soon anyway?